app/ui/src/main/java/com/fsck/k9/ui/folders/FolderIconLoader.kt

```diff
@@ -0,0 +1,60 @@
+package com.fsck.k9.ui.folders
+
+import android.content.res.Resources
+import android.util.TypedValue
+import com.fsck.k9.mail.Folder
+import com.fsck.k9.mailstore.FolderType
+import com.fsck.k9.ui.R
+
+class FolderIconLoader(private val theme: Resources.Theme) {
```

Comment:
This class isn't loading icons when its public methods are used. `FolderIconProvider` would probably be a better name.